src/Monolog/Handler/AbstractBrowserHandler.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Monolog\Handler;
+
+abstract class AbstractBrowserHandler extends AbstractProcessingHandler
```

Comment:
This could be a trait instead, that way it doesn't force the inheritance chain to include AbstractProcessingHandler in case another handler could make use of this method.

Reply:
Done 👍